src/main/resources/models/tables/PlayerStats.hjson

```diff
@@ -0,0 +1,56 @@
+{
+    tables: [{
+        name: PlayerStats
+        table: PlayerStats
+        description:
+        '''
+        A long description
+        '''
+        cardinality : large
+        readAccess : Prefab.Role.All
+        joins: [
+            {
+                name: playerCountry
+                to: PlayerCountry
+                type: toOne
+                definition: '%join.id = %from.countryId'
+            }
+        ]
+        measures : [
+            {
+            name : highScore
+            type : INTEGER
+            definition: 'MAX(highScore)'
+            readAccess : Prefab.Role.All
```

Comment:
we don't need to pass readAccess for every column if its same as the readAccess at table level.